storage/src/main/java/com/brandwatch/ahmedanalytics/storage/Config.java

```diff
@@ -20,8 +20,8 @@ public ObjectMapper objectMapper() {
         return new ObjectMapper();
     }
 
-    @Bean
-    public KafkaConsumer<String, Mention> mentionConsumer(@Value("${spring.kafka.producer.bootstrap-servers}") String bootstrapServers) {
+    private KafkaConsumer<String, Mention> mentionConsumer(@Value("${spring.kafka.producer.bootstrap-servers}") String bootstrapServers,
```

Comment:
Pay attention that you don't need `@Value("${spring.kafka.producer.bootstrap-servers}")` anymore, as you have control over this method, spring is not injecting the value. The value is injected in ` StorageConsumerRunnable consumerThread` method